API-Examples/2024-11-01/knownValidationErrorsList.md

```diff
@@ -0,0 +1,62 @@
+<img width="200" height="37" src="../../images/Gematik_Logo_Flag_With_Background.png"/> <br/>
+
+# List of known validation errors 
+
+This folder contains examples that throw errors when validated with a validator. Below is the list as well as the reason related to the specific file.
+
+Please note that the list only includes XML and JSON files.
+
+| **File** | **Reason** |
+|------|--------|
+|[authentisieren/01_request_ReadCardCertificate.xml](authentisieren/01_request_ReadCardCertificate.xml)|Not a FHIR resource|
```

Comment:
Ich würde "Not a FHIR resource" nicht auflisten, da sie ja nicht unbedingt invalide sind.